src/OSPSuite.UI/Views/Importer/ImporterView.cs

```diff
@@ -70,12 +70,21 @@ private void changeTabs(IXtraTabPage page, IXtraTabPage prevPage)
             prevPage.Appearance.Header.Font = Fonts.NonSelectedTabHeaderFont;
       }
 
+      public override void InitializeResources()
+      {
+         base.InitializeResources();
+         layoutItemSelectRange.AdjustLongButtonSize();
+         layoutItemImportAll.AdjustButtonSize();
+         layoutItemImport.AdjustButtonSize();
+         btnSelectRange.InitWithImage(ApplicationIcons.PreviewOriginData, Captions.Importer.PreviewExcelData);
+         resetImportButtonText();
```

Comment:
This was actually NEVER called when creating the view. e.g icons were not set properly. Never realized it